src/components/visualizations/chart/highcharts/plugins/dataLabelsColors.ts

```diff
@@ -37,6 +39,27 @@ function setLabelsColor(chart: any) {
     });
 }
 
+function whiteIsNotContrastEnough(color: string) {
+    const { R, G, B } = parseRGBColorCode(color);
+    const colorLuminance = calculateLuminance(R, G, B);
+    const WHITE_LUMINANCE = 1;
+    const CONTRAST_TRESHOLD = 0.4;
+    return (WHITE_LUMINANCE - colorLuminance) < CONTRAST_TRESHOLD;
+}
+
+function setContrastLabelsColor(chart: any) {
+    const points = flatMap(getVisibleSeries(chart), (series: any) => series.points)
```

Comment:
we could encap this to a function, let's say `getPointsWithLabel` and reuse from both